src/frontend/screens/PhotosModal/index.tsx

```diff
@@ -1,24 +1,50 @@
-// @flow
-import React, { useState } from "react";
+import React, { useEffect, useRef, useState } from "react";
 import { Dimensions, View, StyleSheet } from "react-native";
 import { TabView } from "react-native-tab-view";
 
 import IconButton from "../../sharedComponents/IconButton";
+import Button from "../../sharedComponents/Button";
 import { CloseIcon } from "../../sharedComponents/icons";
 import { filterPhotosFromAttachments } from "../../lib/utils";
 import useObservation from "../../hooks/useObservation";
 import useDraftObservation from "../../hooks/useDraftObservation";
 import PhotoView from "../../sharedComponents/PhotoView";
 import api from "../../api";
 import type { NavigationProp } from "../../types";
-
+import { useBottomSheetModal } from "../../sharedComponents/BottomSheetModal";
+import { ConfirmDeleteModal } from "./ConfirmDeleteModal";
 const PhotosModal = ({ navigation }: { navigation: NavigationProp }) => {
   const observationId = navigation.getParam("observationId");
   const [index, setIndex] = useState(navigation.getParam("photoIndex") || 0);
   const [{ observation }] = useObservation(observationId);
   const [{ photos: draftPhotos }] = useDraftObservation();
+  const [showHeader, setShowHeader] = useState(false);
+  const { sheetRef, openSheet, closeSheet } = useBottomSheetModal({
+    openOnMount: false,
+  });
+  const modalIsOpen = useRef<boolean>(false);
```

Comment:
curious, but does changing this to state affect the interactions/UI here?

Reply:
Im not sure what you mean. I didn't want to use state here because this check is already linked to the show header state, so there is no reason to attach to state hooks to one state

Reply:
What might be a better pattern, is to have a boolean in the `useBottomSheetModal` which returns true when the modal is open. But it looks like this is not something we can do in the `@gorhom/bottom-sheet`. So we would have to follow the pattern that I used here